spec/controllers/request_game_controller_spec.rb

```diff
@@ -27,6 +27,45 @@
       expect(assigns[:missing]).to eq(1)
     end
 
+    context 'there are no old unclassified requests' do
+
+      before do
+        InfoRequest.destroy_all
+        FactoryGirl.create(:info_request)
+      end
+
+      render_views
+
+      let(:test_url) { help_credits_path(:anchor => "helpus") }
+      let(:site_name) { AlaveteliConfiguration::site_name }
+
+      it 'shows the game homepage' do
+        get :play
+        expect(response).to render_template('play')
+      end
+
+      it 'assigns the game_over template to the flash message' do
+        get :play
+        expect(flash[:notice][:partial]).to eq("game_over.html.erb")
+        expect(flash[:notice][:locals]).to include({
+          :helpus_url => test_url,
+          :site_name => site_name
+        })
+      end
+
+      it 'displays the flash message' do
+        get :play
+        expect(response.body).
+          to include("<p>All done! Thank you very much for your help.</p>")
+
+        expect(response.body).
+          to include("There are <a href=\"#{test_url}\">more things " \
```

Comment:
Here you could use `have_link` and `have_content` if you decide to go down the capybara route.